pkg/apis/extensions/v1beta1/types.go

```diff
@@ -286,6 +286,12 @@ type DeploymentStatus struct {
 
 	// Total number of non-terminated pods targeted by this deployment that have the desired template spec.
 	UpdatedReplicas int32 `json:"updatedReplicas,omitempty"`
+
+	// Total number of pods targeted by this deployment that are ready.
+	ReadyPods int32 `json:"readyPods,omitempty"`
```

Comment:
We also need to be consistent with the names. We have `Replicas` and `UpdatedReplicas`, so these should be `ReadyReplicas` and `NotReadyReplicas` (or the other way around).